prow/cmd/deck/static/user-script.js

```diff
@@ -2,7 +2,7 @@
 
 function createXMLHTTPRequest(fulfillFn, errorHandler) {
     const request = new XMLHttpRequest();
-    const url = "http://localhost:8888/user-data.js";
+    const url = "https://prow.k8s.io/user-data.js";
```

Comment:
Should be configurable

Reply:
I made the url relative which should be /user-data.js for now. Do you think we need to make that configurable as well?

Reply:
shouldn't we rename this script and the endpoint?